seeq/addons/correlation/_seeq_worksheet_writer.py

```diff
@@ -214,7 +214,13 @@ def worksheet_corrs_and_time_shifts(signal_pairs_ids: list, workbook_id: str,
                             condition_id,
                             overwrite,
                             api_client)
-    return "/".join([spy.client.host.replace('/api', ""), 'workbook', workbook_id, 'worksheet', existing_worksheet.id])
+
+    seeq_url_ = get_seeq_url()
+    
+    if seeq_url_ is None:
```

Comment:
We shouldn't ask for user input here. This is a backend function that the UI calls. At this point we should know the Seeq base URL (we asked during installation). This is where I think we could save the URL in a file during installation and use it here if needed. 

This is the idea in pseudo-code:
In `__main__.py`:

```html
if seeq_url` is None:
    seeq_url = input()
	save_to_file(seeq_url) # can be pickle to avoid user temptation to modify it
```


Once we are here in _seeq_worksheet_writer.py:
```html
if seeq_url` is None:
	seeq_url = read_from_file(seeq_url) # can be pickle to avoid user temptation to modify it
```